examples/bundle/pom.xml

```diff
@@ -138,28 +138,6 @@
                                     <goal>restore-instrumented-classes</goal>
                                 </goals>
                             </execution>
```

Comment:
Wouldn't be safer to keep this?

Reply:
The reason why I removed it is that we have a tight coupling of these tests with the core implementation. We do have the codecov check which makes this redundant, but the codecov dashboard gives good insights what has to be covered.